.github/workflows/olbase.yaml

```diff
@@ -12,14 +12,15 @@ jobs:
   docker:
     if: github.repository_owner == 'internetarchive'
     runs-on: ubuntu-latest
+
+    env:
+      TAG_NAME: ${{ github.ref == 'refs/heads/master' && 'latest' || github.ref_name }}
```

Comment:
Can you add a comment of why we're doing this?
Is the idea that we might trigger this on non-master builds and then it would have a tag of the branch name? I guess this is just for testing purposes.